### ISSUE TITLE
add nbactions

### ISSUE THREAD

**Issue by dukescript** [openjfx/javafx-maven-archetypes#7]
Would be nice to add nbactions.xml, so clicking the run button actually starts the application in NetBeans. Something like this:

```
<?xml version="1.0" encoding="UTF-8"?>
<actions>
    <action>
        <actionName>run</actionName>
        <displayName>Run</displayName>
        <goals>
            <goal>clean</goal>
            <goal>javafx:run</goal>
        </goals>
    </action>
</actions>
```

**Comment by geertjanw** [comment]
Yes, we want to do this on the NetBeans side, i.e. by first extending the API: https://issues.apache.org/jira/browse/NETBEANS-3104, since not everyone using the javafx-maven-archetypes makes use of NetBeans.

**Comment by errael** [comment]
The fork errael/javafx-maven-archetypes modifies pom.xml for debug execution and creates nbactions.xml for project with run and debug actions.

**Comment by JaroslavTulach** [comment]
I guess the request has now been addressed by integrating #10

**Comment by errael** [comment]
> I guess the request has now been addressed by integrating #10

No. 

#10 does not include `nbactions.xml`. Comment https://github.com/openjfx/javafx-maven-archetypes/issues/7#issuecomment-565726866 suggests that it will be handled on the NetBeans side.

In addition, the `pom.xml` created by #10 is out of date compared to the one created by the NetBeans specific archetypes which handle profiling: `<execution><id>ide-profile ...` and jlink

**Comment by JaroslavTulach** [comment]
I see:
* Too bad #10 is out of date. 
* Alas, `nbactions.xml` aren't welcomed contribution by other projects. NetBeans probably will have to deal with it some other way.

CCing @sdedic.

**Comment by johanvos** [comment]
What would be the drawback of adding `nbactions.xml` here? As long as it is not breaking other IDE's or CLI deployment, I think it would be a good addition -- but I might miss something?
